release_notes/42-release-notes.adoc

```diff
@@ -81,7 +85,7 @@ For more information, see xref:../operating/use-system-health-dashboard.adoc#vie
 
 Based on testing with a 50-cluster and 4900-node-environment, these improvement includes:
 
-* *2x* speed improvement in hardware restart.
+* *2x* speed improvement in cold start.
```

Comment:
What is a cold start? I've never heard this terminology before (except in relation to a car).

Can this be elaborated some to describe what's being talked about? I can see in the original issue that there's a discussion about a hardware reset and a cold start being different things -- I understand a hardware reset, but do not understand what's stated here.

Am I just missing something fundamental?

Reply:
Thank you @kowen-rh 

I think the intended audience understands the cold start terminology. It is different than a hardware restart and applies to a container. Here is what I found on IBM dev docs at https://developer.ibm.com/articles/reducing-cold-start-times-in-knative/#scale-from-zero-options

> Cold start: When you are creating a pod from a new image for the first time or after a long time has passed since your last use of this pod in the Kubernetes node. During a cold start, the container image(s) are not cached on the node. The image(s) therefore need to be pulled from an external image repository. Once obtained, the container must be started from scratch. 

I'm keeping this as is.